GameFiles/_integration.gsc

```diff
@@ -38,13 +34,35 @@ init()
 
     level.clientCommandCallbacks = [];
     level.clientCommandRusAsTarget = [];
+    level.overrideMethods = [];
+
+    wait 0.1;
+    InitializeGameMethods();
+
+    [[level.overrideMethods["SetDvarIfUninitialized"]]]( level.eventBus.inVar, "" );
+    [[level.overrideMethods["SetDvarIfUninitialized"]]]( level.eventBus.outVar, "" );
+    [[level.overrideMethods["SetDvarIfUninitialized"]]]( "sv_iw4madmin_integration_enabled", 1 );
+    [[level.overrideMethods["SetDvarIfUninitialized"]]]( "sv_iw4madmin_integration_debug", 0 );
+
+    level.logger = spawnstruct();
+    InitializeLogger();
+
+    //The gamename dvar seems to be an IW only thing..
+    if (level.eventBus.gamename == "") {
+        version = getDvar( "version" );
+        switch( version )
+	    {
+            case "Call of Duty Multiplayer - Ship COD_T5_S MP build 7.0.189 CL(1022875) CODPCAB-V64 CEG Wed Nov 02 18:02:23 2011 win-x86":
+                level.eventBus.gamename = "T5MP";
+            break;
+        }
```

Comment:
format: spacing/indention is inconsistent here. Should be 4 spaces for every indent level, no tabs.
VS code should able to convert from tabs to spaces to make it easier if you're using tabs anywhere